### PR TITLE
Add Github Action to push gems

### PR DESCRIPTION
**Pull request by mzumi**
This PR is enable to push gems automatically.
If it is merged, you won't need to push gem manually.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,15 +13,22 @@ configurations {
...
+ }()
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
     compile  "org.embulk:embulk-core:0.9.17"
     provided "org.embulk:embulk-core:0.9.17"
-    compile group: "com.facebook.business.sdk", name: "facebook-java-business-sdk", version: "16.0.2"
+    compile group: "com.facebook.business.sdk", name: "facebook-java-business-sdk", version: "15.0.0"
```
Comment (u110) [inline build.gradle]: Is this change necessary?

Comment (mzumi) [reply build.gradle]: Thanks for your comment.
The change is unnecessary...
I fixed it.  [10b47a2](https://github.com/trocco-io/embulk-input-facebook_ads_insights/pull/19/commits/10b47a2ee5c101938daa2e3fec9ef35a86d73686)

**Review by u110 — APPROVED** [review]
LGTM
